scripts/deployments/steps/06_pool.ts

```diff
@@ -119,6 +122,31 @@ export const step_06 = async (verify = false) => {
       );
     }
 
+    const accountFactory =
+      (await getContractAddressInDb(eContractid.AccountFactory)) ||
+      (await deployAccountFactory(zeroAddress()));
```

Comment:
if we assume that AccountFactory is already deployed then we can use getAccountFactory instead